test/acceptance_with_go_client/search_test.go

```diff
@@ -305,6 +305,65 @@ func TestNearVectorAndObjectAutocut(t *testing.T) {
 	})
 }
 
+func TestHybridExplainScore(t *testing.T) {
+	ctx := context.Background()
+	c := client.New(client.Config{Scheme: "http", Host: "localhost:8080"})
+	c.Schema().AllDeleter().Do(ctx)
+	className := "ParagraphWithManyWords"
+
+	AddClassAndObjects(t, className, string(schema.DataTypeTextArray), c, "text2vec-contextionary")
+	creator := c.Data().Creator()
+	creator.WithClassName(className).WithProperties(
+		map[string]interface{}{"contents": []string{
+			"specific",
+			"hybrid",
+			"search",
+			"object",
+		}, "num": 4}).Do(ctx)
+	defer c.Schema().ClassDeleter().WithClassName(className).Do(ctx)
+
+	t.Run("hybrid explainscore 1", func(t *testing.T) {
+		results, err := c.GraphQL().Raw().WithQuery(fmt.Sprintf("{Get{%s(hybrid:{query:\"rain nice\", fusionType: rankedFusion, alpha: 0.5, properties: [\"contents\"]}){num _additional { score explainScore id }}}}", className)).Do(ctx)
+
```

Comment:
AFAIK the newest go client can do this with the builder pattern - I assume you copied this from another test that was written when the go client did not support this yet

Reply:
Will fix in another patch